frontend/src/app/acrobatics/single-acrobatic/single-acrobatic.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-single-acrobatic',
```

Comment:
**suggestion:** one would typically call such a component `ShowAcrobaticComponent`

Reply:
Renamed to ShowAcrobaticComponent.